Accordion.js

```diff
@@ -75,19 +72,31 @@ class Accordion extends Component {
 
     return (
       <View {...viewProps}>
-      {this.props.sections.map((section, key) => (
-        <View key={key}>
-          <TouchableHighlight onPress={() => this._toggleSection(key)} underlayColor={this.props.underlayColor}>
-            {this.props.renderHeader(section, key, this.state.activeSection === key)}
-          </TouchableHighlight>
-          <Collapsible collapsed={this.state.activeSection !== key} {...collapsibleProps}>
-            {this.props.renderContent(section, key, this.state.activeSection === key)}
-          </Collapsible>
-        </View>
-      ))}
+        {this.props.sections.map((section, key) =>
+          <View key={key}>
+            <TouchableHighlight
+              onPress={() => this._toggleSection(key)}
+              underlayColor={this.props.underlayColor}
+            >
+              {this.props.renderHeader(
+                section,
+                key,
+                this.state.activeSection === key
+              )}
+            </TouchableHighlight>
+            <Collapsible
+              collapsed={this.state.activeSection !== key}
+              {...collapsibleProps}
+            >
+              {this.props.renderContent(
+                section,
+                key,
+                this.state.activeSection === key
+              )}
+            </Collapsible>
+          </View>
+        )}
       </View>
     );
   }
 }
-
-module.exports = Accordion;
```

Comment:
If you've previously done `require('react-native-collapsible')` you need to change it to `require('react-native-collapsible').default` with this export or just use a regular `import` statement. I've had it as `import` in the readme for quite some time now, so I think it's OK to move on, just wanted to make sure this breaking change is intentional and that we communicate it.